packages/next-on-pages/docs/caching.md

```diff
@@ -0,0 +1,52 @@
+# Caching and Data Revalidation
+
+`@cloudflare/next-on-pages` comes with support for data revalidation and caching for fetch requests. This is done in our router and acts as an extension to Next.js' built-in functionality.
+
+## Storage Options
+
+There are various different bindings and storage options that `@cloudflare/next-on-pages` supports for caching.
```

Comment:
I would probably try to mention upfront that this relies on naming conventions (I think it is made clear in the next sections but I'd still clarify it here 🤔)

something like this:
```suggestion
There are various different bindings and storage options that `@cloudflare/next-on-pages` supports for caching. Besides the default behavior implemented via the [Workers Cache API](https://developers.cloudflare.com/workers/runtime-apis/cache/) the other options are based on bindings naming conventions, meaning that providing that they get enabled when bindings with the appropriate names are detected.
```

it's just an idea I'm ok with you don't want to.